src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceCodeAction.cs

```diff
@@ -58,25 +56,6 @@ private static ImmutableArray<CodeActionOperation> CreateRenameOperations(MoveTo
 
             var operations = PooledObjects.ArrayBuilder<CodeActionOperation>.GetInstance();
             operations.Add(new ApplyChangesOperation(moveToNamespaceResult.UpdatedSolution));
```

Comment:
Do we still need the CreateRenameOperations and all of that or can this just use the more sane CodeAction APIs?

Reply:
(minimally inline this now?)